packages/core/src/external/carequality/dq/process-inbound-dq.ts

```diff
@@ -77,3 +80,21 @@ async function getDocumentContents(cxId: string, patientId: string): Promise<str
   }
   return documentContents;
 }
+
+function ccdExists(extrinsicObjects: string[]) {
```

Comment:
nit: `ccdExists` > `containsCcd` ?

Reply:
I feel like this approach to determining if we have a ccd is kinda overkill. Why not just have ccd vs cda metadata files have a different extension when we upload them. Then no need to do all this parsing which adds unnessecary processing time right?

Reply:
Added this as a TODO on the code and into a new issue - [link](https://github.com/metriport/metriport-internal/issues/1905)